src/cargo/ops/cargo_rustc/output_depinfo.rs

```diff
@@ -70,7 +70,14 @@ pub fn output_depinfo<'a, 'b>(context: &mut Context<'a, 'b>, unit: &Unit<'a>) ->
     let mut deps = HashSet::new();
     let mut visited = HashSet::new();
     let success = add_deps_for_unit(&mut deps, context, unit, &mut visited).is_ok();
-    let basedir = None; // TODO
+    let basedir_string;
+    let basedir = match context.config.get_string("build.dep-info-basedir")? {
```

Comment:
Oh actually this may want to use `get_path` as I think it automatically does path conversion and handle relative paths, sorry forgot about that earlier!

Reply:
Oh, nice! I saw that but since Raph's code didn't deal in paths I didn't use it. I'll rework it to use `get_path`.